vis/test/component/README.md

```diff
@@ -0,0 +1,2 @@
+# Component tests
+Isolated stand alone tests for components using https://enzymejs.github.io/enzyme/ .
```

Comment:
Enzyme turned out to be too weak (no DOM events, deep rendering too complicated etc.), so I started using `react-dom` with `react-dom/test-utils` for rendering and `redux-mock-store` for store mocking.

Reply:
:+1: